### DISCUSSION TITLE
How to make background of geotiff transparent in geonode

### DISCUSSION THREAD

**Discussion by TheNumenorean** [corteva/rioxarray#831] [Q&A]
I am trying to generate geotiffs from point data, and generally it is working fine. However, whenever i upload the result to Geonode, the background is rendered black:
![image](https://github.com/user-attachments/assets/b8814201-9495-4f6a-ab81-42eb8b0c87cf)

How do i make the NaN portions transparent?

What im trying:
```
out_grd = make_geocube(
    vector_data=gdf, measurements=["elevation"], resolution=(-0.1, 0.1)
)["elevation"]
out_grd.rio.to_raster(output_path)
```

Sample data:
![image](https://github.com/user-attachments/assets/5b6ce4e2-56df-48b2-9f39-bbf431143d58)

**Comment by RichardScottOZ** [comment]
Have to set the right nodata value.

**Reply by fjmacagno** [reply]
How do i figure out what that is? I have tried -9999 without luck, and my research hasn't brought up anything very useful.

**Comment by RichardScottOZ** [comment]
Look at the representation of the dataset unmasked - look at a histogram, view the dataset in some software and click on the area that you want to know...

>

**Reply by TheNumenorean** [reply]
Looking at my dataset itself was not helpful, it simply showed what i already knew, which was that outside of the tiff had no data at all, and the area where it was black had the value nan or -9999, depending on what i set the nodata value to. However, that did make me think to check the values of a tiff i got elsewhere and worked as expected: it had the value 0 for the transparent areas inside the bounding box. Using that for nodata worked.

I also found in geoserver that you can see the band info by looking at the layer details: 
![image](https://github.com/user-attachments/assets/b861feba-b998-4043-83ed-25c81527fdda)
